independent-projects/arc/runtime/src/main/java/io/quarkus/arc/ContextCreator.java

```diff
@@ -7,7 +7,10 @@
  */
 public interface ContextCreator {
 
+    public static final String KEY_CURRENT_CONTEXT_FACTORY = "io.quarkus.arc.currentContextFactory";
```

Comment:
`public static final` is redundant when declared on an interface.

Reply:
Ah, of course...